parser-library/parse.cpp

```diff
@@ -233,7 +235,9 @@ bool parse_resource_table(bounded_buffer *sectionData,
     return true; // This is not a hard error. It does happen.
   }
 
-  for (i = 0; i < rdt.NameEntries + rdt.IDEntries; i++) {
+  for (std::uint32_t i = 0;
```

Comment:
I have reduced the scope of the counter since it was not used outside of the loop; I've also surrounded the loop condition with a cast to avoid warnings.